packages/flutter/lib/src/material/about.dart

```diff
@@ -372,13 +376,19 @@ class AboutDialog extends StatelessWidget {
   /// Defaults to nothing.
   final List<Widget>? children;
 
+  /// A theme for customizing the backgroundColor, elevation, shape of a dialog.
+  final DialogTheme? dialogTheme;
```

Comment:
Rather than passing a theme, can we not look up the Theme.of(context).dialogTheme?